actions/find_ip.py

```diff
@@ -0,0 +1,22 @@
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from lib.aci import ACIBaseActions
+
+
+class findIP(ACIBaseActions):
+    def run(self, apic="default", credentials=None, ip=None):
```

Comment:
the apic field is marked as `required: true` in the action metadata, yet you set it to a default string here in the python, need to pick one. Either set a default in the action metadata or remove the default in the python file. Similarly, if `ip` is marked required in the action metadata, you should remove the default here in the python file.